### ISSUE TITLE
Create small OpenCv Test

### ISSUE THREAD

**Issue by ashleyredzko** [Mousers/mousetrap#110]
Create or find a small OpenCV test to understand how the camera initiated and displayed to the screen.

**Comment by ashleyredzko** [comment]
Was in the CS Lab with Zane working on this.

Used the tutorial script (capture video) located at:
http://docs.opencv.org/trunk/doc/py_tutorials/py_gui/py_video_display/py_video_display.html#display-video

These tests are entirely in Fedora 21 with Python3 and using the OpenCV installation instructions in the tutorial located here:
https://gist.github.com/collaj/e75c2f48f69e4e9dacdc

Was receiving the following errors:
V4L/V4L2: VIDIOC_S_CROP
Failed to load OpenCL Runtime

OpenCV Error: Unspecified error (The function is not implemented. Rebuild the library with Windows, GTK+ 2.x or Carbon support. If you are on Ubuntu or Debian, install libgtk2.0-dev and pkg-config, then re-run cmake or configure script) in cvShowImage, file /home/mousetrap/opencv/modules/highgui/src/window.cpp, line 522
Traceback (most recent call last):
  File "opencvtest.py", line 17, in <module>
    cv2.imshow('frame',gray)
cv2.error: /home/mousetrap/opencv/modules/highgui/src/window.cpp:522: error: (-2) The function is not implemented. Rebuild the library with Windows, GTK+ 2.x or Carbon support. If you are on Ubuntu or Debian, install libgtk2.0-dev and pkg-config, then re-run cmake or configure script in function cvShowImage

After some extensive research, I found that this was due to OpenCL being used with OpenCV. 

Kevin's blog post listed in the gist above (https://blog.kevinbrown.in/programming/2014/09/27/building-and-installing-opencv-3.html) installs using the following command: 
cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=$(python3 -c "import sys; print(sys.prefix)") -D PYTHON_EXECUTABLE=$(which python3) ..

I found that there is an option to turn OpenCL off. The modified command is:
cmake -D CMAKE_BUILD_TYPE=RELEASE -D WITH_OPENCL=OFF -D CMAKE_INSTALL_PREFIX=$(python3 -c "import sys; print(sys.prefix)") -D PYTHON_EXECUTABLE=$(which python3) ..

This fixed the OpenCL error and V4L error in my test script and also fixed the OpenCL error I was getting when running mousetrap too! Now it's back to just a jumbled video feed instead of just displaying "Failed to load OpenCL Runtime"

However, I am still getting the video feed jumbling and my test script still has the third error.

Will be testing more when I can. Spoke with Stoney and he mentioned we use GTK to display the video feed, not opencv, so will toy around with that.
